proposals/NNNN-max-records-per-node.md

```diff
@@ -0,0 +1,116 @@
+<!-- {% raw %} -->
+
+* Proposal: [MaxRecordsPerNode Attribute for NodeArrayOutput](NNNN-filename.md)
+* Author(s): [Anupama Chandrasekhar](https://github.com/anupamachandra), [Mike Apodaca](https://github.com/mapodaca-nv)
+* Sponsor: TBD
```

Comment:
Before we merge the PR, we need to find a `Sponsor`.  You'll want to update the number `NNNN` to the next number just before merging as well.

Reply:
What's the process to get a Sponsor assigned?

Reply:
Feel free to put me down as sponsor.